addons/event_sale/models/event_ticket.py

```diff
@@ -18,7 +18,7 @@ def _default_product_id(self):
     # product
     product_id = fields.Many2one(
         'product.product', string='Product', required=True,
-        domain=[("event_ok", "=", True)], default=_default_product_id)
+        domain=[("type", "=", "event")], default=_default_product_id)
```

Comment:
Shouldn't you update ``_init_column`` that creates a missing product at update time, by the way ?

Reply:
Isn't this handled by the upgrade script?